src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -208,25 +208,26 @@ String getAppPath(String appName) {
      */
     @Override
     String processClassPath(String classPath) throws IOException {
-        if (!this.projectConfiguration.isUseJavaFX()) {
+        if (!projectConfiguration.isUseJavaFX()) {
             return classPath;
         }
         // we are using JavaFX
-        String javafxSDK = FileDeps.getJavaFXSDK(projectConfiguration).resolve("lib").toString();
+        String javafxSDKLibs = FileDeps.getJavaFXSDKLibsPath(projectConfiguration).toString();
```

Comment:
`javafxSDKLibs` can be a `Path`, which makes it easy to build subpaths out of (using resolve) later

Reply:
ok, refactored